config/app.default.php

```diff
@@ -473,4 +473,18 @@
     'Pagination' => [
         'sizeAvailable' => [10, 20, 50, 100]
     ],
+
+    /**
+     * Upload default settings
+     *
+     * - models => available models (value => label) to associate to upload
+     */
+    'Upload' => [
```

Comment:
These types can be obtained using API to avoid configuration.
Proposal: let's use `images`, `audio`, `videos` and `files` as default set, in code not in config - later via API

Then looking at `/home` response used to build modules list we may see if these types are actually available and if user has permissions to create/modify looking in `meta.resources.{endpoint}.hints.allow`